projects/swaypad/src/main.sw

```diff
@@ -1,9 +1,6 @@
 contract;
 
 abi TestContract {
-    #[storage(write)]
```

Comment:
Thanks for the contribution! For this to be updated in the default example, we need to set it in `constants.ts` as well!